wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -0,0 +1,205 @@
+- **La normalización del desplazamiento se debe hacer en beatmaps con BPM variable.**
+
+#### Pautas
+
+- **Valores del HP y OD no deben ser más grandes que 7.**
+- **Snaps consecutivos de 1/4 o más no deberían ser usados.** Ritmos lentos son más apropiados para este nivel de dificultad.
+- **Las notas largas deben ser presionadas por al menos una pulsación.** Soltar la nota más rápido que esto prueba ser difícil para jugadores de este nivel.
+- **El final o release de las notas largas deben estar al menos a una pulsación separada de cada una.** Brechas más pequeñas que esto requieren un nivel de independencia de los dedos que estos jugadores de esta dificultad no pueden manejar.
+- **Objetos no deben de ser puestos mientras una nota larga de 1/1 se presiona.** Sin embargo, poner un objeto al principio o final de dicha nota está bien.
+- **La densidad de las notas debería de consistir en mayormente 1/1, 1/2 ocasional, o ritmos más lentos.**
+- **Evite usar más de 5 notas consecutivas de 1/2.**
+- **Evite usar chords durante patrones de notas de 1/2 consecutivos.** Sin embargo, usar un chord al inicio o final de un patrón está bien.
+- **Evite usar chords con más de 2 notas.** Esto incluye patrones de notas largas que involucren notas en más de dos columnas.
+
+### ![](/wiki/shared/diff/normal-m.png?20211215) Normal
+
+#### Reglas
+
+Si una dificultad Normal es requerida y usada como la *dificultad más baja* de un beatmap, también debe de cumplir con estas reglas:
+
+- **Los cambios de la velocidad de deslizamiento que alteren la velocidad de un beatmap están prohibidos.** Los cambios de velocidad que unifican el deslizamiento de una canción con BPM variable están exentos.
+- **La normalización del desplazamiento se debe hacer en beatmaps con BPM variable.**
+
+#### Pautas
+
+- **Valores del HP y OD no deben ser más grandes que 7.5.**
+- **Notas con un snap consecutivo de 1/6 y más no deberían ser usados.** Ritmos lentos son más apropiados para este nivel de dificultad.
+- **Las notas largas deben ser presionadas por al menos 1/2 pulsación.** Soltar la nota más rápido que esto prueba ser difícil para jugadores de este nivel.
+- **El final o release de las notas largas deben estar al menos a 1/2 pulsación separada de cada una.** Brechas más pequeñas que esto requieren un nivel de independencia de los dedos que estos jugadores de esta dificultad no pueden manejar.
+- **Objetos no deben de ser puestos mientras una nota larga de 1/2 se presiona.** Sin embargo, poner un objeto al principio o final de dicha nota está bien.
+- **Evite usar ritmos más rápidos que 1/2 mientras que una nota larga de 1/1 se presiona.**
+- **Evite usar cualquier tipo de patrón de shield de 1/4.** La coordinación requerida para este patrón es muy alta para jugadores de este nivel.

```

Comment:
Added "los", kept the rest of it as is since that wording can feel confusing since "coordinación" should be represented by an adverb of manner rather than a plain adjective due to it being on a higher skill cap for players of this level, this could deviate the user of thinking about another topic when we are still talking about the same one.